### DISCUSSION TITLE
Read-only HDD

### DISCUSSION THREAD

**Discussion by RetromanXGames** [retronas/discussions#27] [Q&A]
I can't copy or edit files on my HDD, even accessing retronas from another PC, I even took the hd out of the pc with retronas and put it in a case and test it on another machine and I can copy and paste, edit files quietly, if I put it in retronas I always get a read-only message
![1](https://user-images.githubusercontent.com/74443668/212471325-5977451f-8126-457a-b3c9-0e2aaed533f2.png)
![2](https://user-images.githubusercontent.com/74443668/212471326-474f4838-de04-4c86-b76e-efcbb58aad35.png)

Even with the recording options selected in the file, the error persists:
![4](https://user-images.githubusercontent.com/74443668/212471477-ba6f7255-cd2c-4fcd-b905-6defdaed148e.png)

My fstab configuration for games HDD:
![3](https://user-images.githubusercontent.com/74443668/212471330-4a958ae0-5b73-4ba7-bd38-980d89c8f6c8.png)

Everything else works ok, ps3, ps2 and xbox 360 loading games normally, I just can't add more games or rename files over the network because of this error

**Comment by sairuk** [comment]
ntfs can be problematic when used under Linux, and especially when writing too it. While the Linux support is _there_ i wouldn't recommend using it particularly for writing too.

What is the output of the `mount | grep games` command in the terminal,  i assume the command will show it is mounted `ro` and I'd assume that is due to a dirty bit or some other general error ntfs seems to suffer from, Linux will not mount the drive `rw` if it has been uncleanly mounted to protect the data.

In Cockpit check the Logs sections for additional information if the mount was successful or not,

You will most like need to plug it into a windows machine and run chkdsk, then cleanly disconnect the drive and reconnect to RetroNAS. I don't think we'll support ntfs moving forward, its a closed source product and the Linux support should be considered "best effort"

**Comment by sairuk** [comment]
Converted to Discussion, this is a configuration use case question not a functional issue with RetroNAS

**Comment by RetromanXGames** [comment]
thanks for moving to the correct section and sorry for posting in the wrong place.

here's what pops up after i type the command, it's actually with ro
![1](https://user-images.githubusercontent.com/74443668/212496026-289d66bc-7c79-48ee-9216-829ff7cda2b3.jpeg)

in the log there really appears an error in fstab
![1](https://user-images.githubusercontent.com/74443668/212496525-8c3da2b9-02b8-4f69-80c2-68790f396609.jpeg)

So the only way to fix this would be to format it on another system?

**Reply by sairuk** [reply]
What does you fstab entry look like? `grep games /etc/fstab` we can see if its correct or if there is actually a duplication.



<blockquote>
So the only way to fix this would be to format it on another system?
</blockquote>

ultimately I'd say yes, you can use something like ext2, 3 or 4 which is opensource and can still be read on windows with a driver. While you could probably workaround it for now you should expect to runinto problems again with NTFS. Although keep in mind once RetroNAS is setup you would just access it over the network from Windows so the underlying filesystem on the RetroNAS side really makes no difference.

for now if you want to try and resolve this with NTFS you can try plugging the disk back into Windows and run a chkdsk on it (linux does have tools for this but I can't recommend them) then cleanly disconnect it with a shutdown and then plug it back into RetroNAS

**Comment by RetromanXGames** [comment]
Thank you very much for the answers, I decided to back up everything and format it to leave it the way you indicated, then I install a driver for windows to be able to view the hd as you said, I hope I solve it, working, then I just do it RGBuntu reads what's on the HD that all the systems I have here will be able to pull retronas games.
